package.js

```diff
@@ -0,0 +1,50 @@
+Package.describe({
+  name: 'peerlibrary:classy-test',
+  summary: "Class-based wrapper around tinytest.",
+  version: '0.1.0',
+  git: 'https://github.com/peerlibrary/meteor-classy-test.git'
+});
+
+Package.onUse(function(api) {
+  api.versionsFrom('METEOR@1.0');
+
+  // Core dependencies.
+  api.use([
+    'underscore',
+    'coffeescript',
+    'tinytest'
+  ]);
+
+  api.export('ClassyTestCase');
+
+  // Client and server.
+  api.addFiles([
+    // TODO: This include is needed as test-helpers only export asyncMulti to tests.
+    'meteor/packages/test-helpers/async_multi.js',
+    // TODO: This include is needed as Tinytest does not export TestCaseResults.
+    // Note that we can't simply import 'meteor/packages/tinytest/tinytest.js' as this
+    // overwrites Tinytest and causes test registration not to work.
```

Comment:
Can you explain what is the problem here? Often you can trick things here. So I think there is some workaround.